markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -44,6 +44,9 @@ contract PerpsAccountModule is IAccountModule {
         globalPerpsMarket.checkLiquidation(accountId);
 
         PerpsAccount.Data storage account = PerpsAccount.load(accountId);
+        if (account.id == 0) {
+            account.addAccountId(accountId);
```

Comment:
can just do `account.id = accountId` and nix the `addAccountId` function

Reply:
I see, will do